statsmodels/tsa/holtwinters.py

```diff
@@ -632,7 +632,7 @@ def fit(self, smoothing_level=None, smoothing_slope=None, smoothing_seasonal=Non
         if np.ndim(y) != 1:
             raise ValueError('Only 1 dimensional data supported')
         self._y = y
-        l = np.zeros(self.nobs)
+        lvals = np.zeros(self.nobs)
```

Comment:
that's a lot of changes in this module, but the name is not more informative, just less ambiguous to read

Reply:
hmm eventually we define `level = l[1:nobs + 1].copy()`.  Could make this something like `lvl` or `level0`

Reply:
Could you s/r lvals -> lvls , then ready.